kube/kd/service.yaml

```diff
@@ -3,12 +3,12 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    name: hocs-case-creator
-  name: hocs-case-creator
+    name: hocs-case-{{.DEPLOYMENT_TYPE}}
+  name: hocs-case-{{.DEPLOYMENT_TYPE}}
 spec:
   ports:
   - name: https
     port: 443
-    targetPort: 10443
```

Comment:
Same here this can stay, as it uses the selector to pick which deployment to send to.

Reply:
Reverted this change.